mlrun/projects/project.py

```diff
@@ -2957,3 +2948,13 @@ def _has_module(handler, kind):
 
 def _is_imported_artifact(artifact):
     return artifact and isinstance(artifact, dict) and "import_from" in artifact
+
+
+def _get_item_absolute_path(url, project):
+    if url and "://" not in url:
+        if project.spec.context and not url.startswith("/"):
+            url = path.join(project.spec.get_code_path(), url)
+            return url, True
```

Comment:
just so it would be clear, I'd change the True/False here to `in_context` and return it. this will help to understand what is the meaning of the True vs False here